README.md

```diff
@@ -143,7 +176,7 @@ avalanche network status
 
 A node needs to catch up to the latest network state before it can participate in consensus and serve API calls. This process (called bootstrapping) currently takes several days for a new node connected to Mainnet.
 
-A node will not [report healthy](https://docs.avax.network/build/avalanchego-apis/health) until it is done bootstrapping.
+A node will not [report healthy](https://docs.avax.network/api-reference/health-api) until it is done bootstrapping.
```

Comment:
this is a helpful suggestion, If you want to open a new PR to make this change, I would be happy to approve it.